graphql_compiler/tests/test_macro_schema.py

```diff
@@ -32,3 +34,27 @@ def test_get_schema_with_macros_basic(self):
             'Animal').fields['out_Animal_RelatedFood'].type
         self.assertTrue(isinstance(related_food_target_type, GraphQLList))
         self.assertEqual('Food', related_food_target_type.of_type.name)
+
+    def test_get_schema_for_macro_definition_unchanged(self):
+        original_schema = self.macro_registry.schema_without_macros
+        original_printed_schema = print_schema(original_schema)
+        macro_definition_schema = get_schema_for_macro_definition(original_schema)
+        macro_definition_printed_schema = print_schema(macro_definition_schema)
+        required_directives = get_required_macro_edge_directives()
+        # pylint: disable=protected-access
+        for directive in required_directives:
+            self.assertTrue(directive in macro_definition_schema._directives)
```

Comment:
Please use get_directives and remove the don't disable pylint.

Reply:
Good point! Fixed in commit "Address comments."